packages/runner/integration/patterns/asset-lifecycle-tracker.ts

```diff
@@ -0,0 +1,210 @@
+import type { PatternIntegrationScenario } from "../pattern-harness.ts";
+import type {
```

Comment:
This is importing a pattern into our Deno runtime directly -- this works due to aliasing `"commontools"` to `"@commontools/api"` in our workspace, but seems like undefined behavior (or will be) to invoke `derive` and other pattern lib methods from within the Deno runtime -- importing types only circumvents this at least

Reply:
this is the test file and it's only sharing types with the pattern (`import _type_`). the .pattern.ts file will run through our runtime. do you think this is still problematic? the imported types are unrelated to the API.